### ISSUE TITLE
Diesel (or steam) and electric locos don't work well together

### ISSUE THREAD

**Issue by Federico57** [voxeltycoon/bug-tracker#147]
If you want to create a train with two locomotives, diesel/steam and electric (the order doesn't matter), you need an electrified line. In fact even if you have a train with 100 diesel locos and only 1 electric power unit, the train is blocked by the latter.
So according to logic, the train should travel as if it were composed of a single locomotive (diesel or steam) and an extra wagon (electric loco).

This bug can be found in every version and all you have to do is to put together two different locomotives, including an electric one, in an unelectrified line.

![Screenshot (111)](https://user-images.githubusercontent.com/52663433/64033285-6c832d80-cb4c-11e9-80ce-b7d91d543b2b.png)

**Comment by Funky7Monkey** [comment]
I fail to see how this is actually a problem. Realistically, yeah, it would be a bit weird if a train with a diesel and an electric locomotive were unable to go on unelectrified track, but this is a game. I personally don't see any reason why anyone should be using unlike locomotives on a train.

**Comment by Federico57** [comment]
You're right, no one would use two different locomotives for a single train. I thought it would be better to report it just before I forgot it. Moreover if it is true that the sorting stations and the locos fuels will be added, perhaps one day this will become a serious problem.

Inviato dal mio dispositivo Huawei

-------- Messaggio originale --------
Oggetto: Re: [voxeltycoon/bug-tracker] Diesel (or steam) and electric locos don't work well together (#147)
Da: Funky7Monkey
A: voxeltycoon/bug-tracker
CC: Fredrik Leftin ,Author


I fail to see how this is actually a problem. Realistically, yeah, it would be a bit weird if a train with a diesel and an electric locomotive were unable to go on unelectrified track, but this is a game. I personally don't see any reason why anyone should be using unlike locomotives on a train.
